packages/react-composites/src/composites/CallComposite/types/ErrorTracking.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ErrorType } from '@internal/react-components';
+
+/** @private */
+export interface ErrorTrackingInfo {
+  mostRecentlyActive: Date;
+  lastDismissedAt?: Date;
+}
+
+/** @private */
+export type TrackedErrors = Partial<Record<ErrorType, ErrorTrackingInfo>>;
```

Comment:
Not requesting a change here but we should think about the typing here. We had a ARB comment recently around types defined as `{ [key: string]: Object }` that is just essentially just a `Record<string, Object>`. do we  know which typing we want to align with to not confuse the ARB? (We currently observe the former)

Reply:
Great point, no idea on this. Do you have a link to the apiview comment? They should update the azure policies if they have a strong preference but I'm also curious what the difference is!
(This is private so at least doesn't matter for this PR but is a good callout to highlight for everyone across the repo)